src/main/java/Arriky.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+public class Arriky {
+    public static void main(String[] args) {
+
+        // initialize
+        Scanner sc = new Scanner(System.in);
+        TaskList tl = new TaskList();
+        greet();
+        boolean running = true;
```

Comment:
Variable should be named as isRunning

Reply:
Thanks for pointing out, I'll take it. Boolean name should be descriptive.